R/group_apply.R

```diff
@@ -0,0 +1,119 @@
+#' Apply a function over grouping(s) of data
+#'
+#' A function to apply `palaeoverse` functionality across subsets (groups) of
+#' data, delineated using one or more variables. `data.frame` related functions
+#' (e.g. `nrow`, `ncol`, `lengths`, `unique`) may also be used.
+#'
+#' @param occdf \code{dataframe}. A dataframe of fossil occurrences.
+#' This dataframe must contain the grouping variables and the necessary
+#' variables for the function you wish to call (see function specific
+#' documentation for required columns).
+#' @param group \code{character}. A vector of column names, specifying the
+#' grouping (e.g. "collection_no", "stage_bin").
```

Comment:
```suggestion
#' desired subgroups (e.g. "collection_no", "stage_bin"). Supplying more than one grouping variable will produce an output containing subgroups for each unique combination of values.
```
Think my wording could be improved here but I think a sentence to this effect is a good idea.